src/main/capi/duckdb_value-c.cpp

```diff
@@ -393,6 +393,91 @@ duckdb_value duckdb_create_array_value(duckdb_logical_type type, duckdb_value *v
 	return WrapValue(array_value);
 }
 
+duckdb_value duckdb_create_map_value(duckdb_logical_type map_type, duckdb_value keys, duckdb_value values) {
+	if (!map_type || !keys || !values) {
+		return nullptr;
+	}
+
+	auto &map_logical_type = UnwrapType(map_type);
+	if (duckdb::TypeVisitor::Contains(map_logical_type, duckdb::LogicalTypeId::INVALID) ||
```

Comment:
I believe this is outdated logic, if it's meant to detect prepared parameters?
See <https://github.com/duckdb/duckdb/pull/16264/files> as a recent example of a way to detect this, this was changed fairly recently but I can't remember what PR this was